src/pages/index.module.scss

```diff
@@ -1,5 +1,17 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap");
 
+@mixin small {
```

Comment:
**nitpick:** Preferably we should name these variables more meaningful - like `mobile`, or `desktop`